accounts/scwallet/wallet.go

```diff
@@ -303,7 +303,7 @@ func (w *Wallet) Status() (string, error) {
 
 	// If the card is not paired, we can only wait
 	if !w.session.paired() {
-		return "Unpaired, waiting for pairing password", nil
+		return "Unpaired, waiting for pairing password (call personal.initializeWallet)", nil
```

Comment:
I don't think the library should make assumptions about the calling code, specifically the exposed RPC methods. If Clef uses this, it won't be in the `personal` namespace. It might be ok to say `call initializeWallet`, at least then the namespace is not assumed.

Reply:
@holiman done